packages/react/src/patterns/sub-patterns/ButtonGroup/README.stories.mdx

```diff
@@ -115,6 +117,11 @@ ReactDOM.render(<App />, document.querySelector('#app'));
 
 <Props of={ButtonGroup} />
 
+| Name      | Required | Data Type | Default Value | Description                                                 |
+| --------- | -------- | --------- | ------------- | ----------------------------------------------------------- |
+| `buttons` | NO       | Array     | null          | Array of button objects to render. See `Button Item` below. |
+| `inverse` | NO       | Boolean   | false         | Changes theme to inverse/default               |
+
```

Comment:
@greeshmareddyg I don't think this is needed, right? Since we're already pulling in the props from the component in the line just above?

Reply:
@kennylam yeah, removed the props description here.